site/docs/components/Icon.tsx

```diff
@@ -0,0 +1,169 @@
+export type IconProps = {
```

Comment:
I think we should handle these SVGs similar to how we organize and access them in `/src` 

Icon folder with each SVG in its own file - `astroSVG.ts` etc

Reply:
There is a` site/docs/components/svgs `folder already https://github.com/coinbase/onchainkit/blob/paul/update-maxSlipapge/site/docs/components/svg/fundSvg.tsx#L1

Reply:
Also try to use `ock-fill-*` variables for styling so the icons will update during theme changes!

Reply:
okay, i'll give this a shot.